apps/server/src/modules/deletion/uc/deletion-request.uc.ts

```diff
@@ -144,10 +143,13 @@ export class DeletionRequestUc {
 		await this.logDeletion(deletionRequest, DomainModel.ACCOUNT, DeletionOperationModel.DELETE, 0, 1);
 	}
 
-	private async removeUserRegistrationPin(deletionRequest: DeletionRequest) {
-		const userToDeletion = await this.userService.findById(deletionRequest.targetRefId);
+	private async removeUserRegistrationPin(deletionRequest: DeletionRequest): Promise<number[]> {
```

Comment:
I think returning just a `number` instead of a `number[]` should be enough here according to the method's logic.

Reply:
done